spec/models/course_spec.rb

```diff
@@ -1678,31 +1679,10 @@
       subject { create(:course, site_statuses: []) }
 
       it "sets the site to new when a new site is added" do
-        expect { subject.add_site!(site: new_site) }.to change { subject.reload.site_statuses.size }.
-          from(0).to(1)
+        expect { subject.sites = [new_site] }.to change { subject.reload.site_statuses.size }.from(0).to(1)
         expect(new_site_status.status).to eq("new_status")
       end
     end
-
-    context "for mixed courses with new and running locations" do
```

Comment:
I've removed this as i do not believe it is possible to have a mixture of new and running `site_statuses`

Reply:
👍